deployment/README.md

```diff
@@ -23,7 +23,7 @@ Amazon Web Services deployment is driven by [Terraform](https://terraform.io/),
 
 #### SSH Keys
 
-You'll need to download the `district-builder` SSH Keypair from the fileshare, and store it at `~/.ssh/district-builder.pem`.
+You'll need to download the `district-builder-pa` SSH Keypair from LastPass, and store it at `~/.ssh/district-builder-pa.pem`.
```

Comment:
FWIW I had a pretty tough time getting this to work; LastPass requires a second plugin, distinct from the standard browser extension, to be installed in order to download files, and even then I still wasn't able to get it to work in Firefox.